final_prep/mod_zero_hero.rb

```diff
@@ -2,51 +2,94 @@
 
+hero_name = "String1"
+special_ability = "String2"
+
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
 
+greeting = "No need to fear, #{hero_name} is here!"
+catchphrase = "You just got #{special_ability}!"
+
 # Declare two variables - power AND energy - set to integers
 
+power = 3
+energy = 7
+
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
 
+full_power = power * 500
+full_energy = energy * 150
+
 # Declare two variables - is_human and identity_concealed - assigned to booleans
 
+is_human = true
+identity_concealed = true
+
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
 
+arch_enimies = ["Mr. Blob", "Cat Herair", "Ugly Duckling"]
+sidekicks = ["Boomshakalaka", "Yoshi", "Mr. Sovereign"]
 # Print the first sidekick to your terminal
+puts sidekicks[0]
 
 # Print the last arch_enemy to the terminal
+puts arch_enimies[-1]
 
 # Write some code to add a new arch_enemy to the arch_enemies array
+arch_enimies.push("Fire Blaster")
 
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
+puts arch_enimies
 
 # Remove the first sidekick from the sidekicks array
+sidekicks.delete_at(0)
 
-# Print the sidekicks array to terminal to ensure you added a new sidekick
-
+# Print the sidekicks array to terminal to ensure you added (type, remove?) a new sidekick
+puts sidekicks
+puts "-" * 10
 # Create a function called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
 #   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
+#   - save_the_day should be a string a hero would say once they save the day
 #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
+def assess_situation(danger_level, save_the_day, bad_excuse)
+
+  save_the_day = "Don't worry. Mr. Hero is here. It'll be okay."
+  bad_excuse = "Cops will take care of it."
+
 
 # Your function should include an if/else statement that meets the following criteria
 #   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the bad_excuse to the terminal
 #   - Anything danger_level that is between 10 and 50 should result in printing the save_the_day string to the terminal
 #   - If the danger_level is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the terminal.
 
+  if danger_level > 50
+    puts bad_excuse
+  elsif danger_level > 10
```

Comment:
What if danger_level == 10? "Meh. Hard pass" should only be printed if the danger_level is below 10.